### ISSUE TITLE
Covid19MapCasesUS()  is not working 

### ISSUE THREAD

**Issue by usback** [winfinit/covid19-us#1]
```
let mapCases = new Covid19MapCasesUS();
mapCases.getCDCMapCasesData().then(data => {
    console.log("maps data:", data);
    /* 
        Sample output:
        [ { 
            casesReported: 0,
            stateCode: 'AL',
            stateName: 'Alabama',
            isEstimated: false 
        }
        // ...
        ]
    */
});
```

error message

---------------------------------------------
(node:37326) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'forEach' of undefined
    at Covid19MapCasesUS.formatCDCMapCasesData (/Volumes/data/project/coronainusa/node_modules/covid19-us/built/src/covid19-map-cases-us.js:73:25)
    at /Volumes/data/project/coronainusa/node_modules/covid19-us/built/src/covid19-map-cases-us.js:93:31
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:37326) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:37326) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
deaths {
---------------------------------------------

thank you ..
how can i fix that

**Comment by winfinit** [comment]
hello, what version of node are you using?

**Comment by winfinit** [comment]
nm, able to duplicate it also now

**Comment by winfinit** [comment]
just published a fix in 1.1.0 release. previous CDC json data feed changed, which required me to switch to a CSV stream.

**Comment by usback** [comment]
i got it ..
 Thanks so much !!
